Say Their Names/Source/Model/Person.swift

```diff
@@ -8,22 +8,51 @@
 
 import Foundation
 
-struct Person: Codable {
-    var id: String
+protocol PersonInterface: Decodable {
+    var id: Int { get set }
+    var fullName: String { get set }
+    var dob: String { get set }
+    var doi: String { get set }
+    var childrenCount: String { get set }
+    var age: String { get set }
+    var city: String { get set }
+    var country: String { get set }
+    var bio: String { get set }
+    var context: String { get set }
+    var images: [Images] { get set }
```

Comment:
Maybe we want to add thumbnail (url) here since we're using this model in the home screen and there we're showing just one image

<img width="183" alt="Screen Shot 2020-06-03 at 22 19 04" src="https://user-images.githubusercontent.com/5240729/83704355-46615100-a5e8-11ea-993b-410b71dbec65.png">

any thoughts?